NOTES.md

```diff
@@ -1,9 +1,60 @@
+# Release 2.1.5 (Forbidden Planet)
+
+Released on 2015-10-07
+
+Bug fix release
+
+### Compatiblity ###
+
+* RethinkDB 2.1.5 servers cannot be mixed with servers running RethinkDB 2.1.1 or earlier
+  in the same cluster
+
+### Bug fixes ###
+
+* Made the build system compatible with OS X El Capitan (#4602)
+* Fixed spurious "Query terminated by `rethinkdb.jobs` table" errors (#4819)
+* Fixed an issue that caused changefeeds to keep failing after a table finished
+  reconfiguring (#4838)
+* Fixed a race condition that resulted in a crash with the message
+  `std::terminate() called without any exception.` when losing a cluster connection
+  (#4878)
+* Fixed a segmentation fault in the `mark_ready()` function that could occurr when
+  reconfiguring a table (#4875)
+* Fixed a segmentation fault when using changefeeds on `orderBy.limit` queries (#4850)
+* Made the Data Explorer handle changefeeds on `orderBy.limit` queries correctly (#4852)
+* Fixed a "Branch history is incomplete" crash when reconfiguring a table repeatedly in
+  quick succession (#4866)
+* Fixed a problem that caused `indexStatus` to report results for additional indexes that
+  were not specified in its arguments (#4868)
+* Fixed a segmentation fault when running RethinkDB on certain ARM systems (#4839)
+* Fixed a compilation issue in the UTF-8 unit test with recent versions of Xcode (#4861)
+* Fixed an `Assertion failed: [ptr_]` error when reconfiguring tables quickly with a
+  debug-mode binary (#4871)
+* Improved the detection of unsupported values in `r.js` functions to avoid a
+  `Guarantee failed: [!key.IsEmpty() && !val.IsEmpty()]` crash in the worker process
+  (#4879)
+* Fixed an unitialized data access issue on shutdown (#4918)
+
+### Performance improvements ###
+
+* Improved the performance of `getAll` queries which fetch multiple keys at once (#1526)
+* Optimized the distribution of tasks across threads on multi-core servers (#4905)
+
+--
+
 # Release 2.1.4 (Forbidden Planet)
 
 Released on 2015-09-16
 
 Bug fix release
 
+### Compatiblity ###
```

Comment:
This is slightly unorthodox, but I decided to retrofit the 2.1.4 release notes to have the same format and compatibility warning as the ones for 2.1.2 and 2.1.3 (and now 2.1.5).